app/screens/share_feedback/index.tsx

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+import {View, Text, TouchableOpacity} from 'react-native';
+
+import {goToNPSChannel} from '@actions/remote/channel';
+import Button from '@components/button';
+import CompassIcon from '@components/compass_icon';
+import SearchIllustration from '@components/no_results_with_term/search_illustration';
+import {useServerUrl} from '@context/server';
+import {useTheme} from '@context/theme';
+import useBackNavigation from '@hooks/navigate_back';
+import {dismissOverlay} from '@screens/navigation';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+type Props = {
+    componentId: string;
+}
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    root: {
+        flex: 1,
+        backgroundColor: changeOpacity('#000000', 0.40),
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+    container: {
+        flex: 1,
+        maxWidth: 680,
+        alignSelf: 'center',
+        alignContet: 'center',
+        alignItems: 'center',
+        justifyContent: 'center',
+        width: '100%',
+        flexDirection: 'row',
+    },
+    wrapper: {
+        backgroundColor: theme.centerChannelBg,
+        borderRadius: 12,
+        flex: 1,
+        margin: 10,
+        opacity: 1,
+        borderWidth: 1,
+        borderColor: changeOpacity(theme.centerChannelColor, 0.16),
+
+    },
+    content: {
+        marginHorizontal: 24,
+        marginBottom: 24,
+        alignItems: 'center',
+    },
+    buttonsWrapper: {
+        flexDirection: 'row',
+        width: '100%',
+    },
+    close: {
+        justifyContent: 'center',
+        height: 44,
+        width: 40,
+        paddingLeft: 16,
+    },
+    title: {
+        ...typography('Heading', 600, 'SemiBold'),
+        color: theme.centerChannelColor,
+        marginTop: 24,
+        marginBottom: 8,
+        textAlign: 'center',
+    },
+    subtitle: {
+        ...typography('Body', 200, 'Regular'),
+        color: changeOpacity(theme.centerChannelColor, 0.72),
+        marginBottom: 24,
+        textAlign: 'center',
+    },
+    leftButton: {
+        flex: 1,
+        marginRight: 5,
+    },
+    rightButton: {
+        flex: 1,
+        marginLeft: 5,
+    },
+}));
+
+const close = (componentId: string) => {
+    dismissOverlay(componentId);
+};
```

Comment:
same as the other file, but I'm not sure why you need this function when you can call `dismissOverlay` directly